galaxy/Dockerfile

```diff
@@ -186,49 +230,54 @@ RUN mkdir -p /shed_tools $EXPORT_DIR/ftp/ \
     && ln -s /tool_deps/ $EXPORT_DIR/tool_deps \
     # Configure Galaxy to use the Tool Shed
     && chown $GALAXY_USER:$GALAXY_USER $EXPORT_DIR/tool_deps \
-    && apt update -qq && apt install --no-install-recommends -y ansible \
+    && apt update -qq && apt install --no-install-recommends -y ansible gpg gpg-agent g++ make bsdmainutils \
```

Comment:
Here ansible is installed again.

Reply:
```suggestion
    && apt update -qq && apt install --no-install-recommends -y gpg gpg-agent g++ make bsdmainutils \
```

We should not need g++, which is something to investigate at the very end.

Reply:
I would check that.

Reply:
I have removed g++